File: jphp-javafx-ext/src/main/resources/JPHP-INF/sdk/ext/javafx/UXTextArea.php
<?php
namespace ext\javafx;

/**
 * Class UXTextArea
 * @package ext\javafx
 */
class UXTextArea extends UXTextInputControl
{
    /**
     * @var bool
     */
    public $wrapText;

    /**
     * @var double
     */
    public $scrollLeft;

    /**
     * @var double
     */
    public $scrollTop;

    /**
     * @var int
     */
    public $prefRowCount;

    /**
     * @var int
     */
    public $prefColumnCount;

    /**
     * @param string $text (optional)
     */
    public function __construct($text) {}
}

File: exts/jphp-gui-ext/src/main/java/org/develnext/jphp/ext/javafx/classes/UXTextArea.java
package org.develnext.jphp.ext.javafx.classes;

import javafx.scene.control.TextArea;
import org.develnext.jphp.ext.javafx.JavaFXExtension;
import php.runtime.annotation.Reflection.Name;
import php.runtime.annotation.Reflection.Property;
import php.runtime.annotation.Reflection.Signature;
import php.runtime.env.Environment;
import php.runtime.reflection.ClassEntity;

@Name(JavaFXExtension.NS + "UXTextArea")
public class UXTextArea extends UXTextInputControl {
    interface WrappedInterface {
        @Property boolean wrapText();
        @Property double scrollLeft();
        @Property double scrollTop();
        @Property int prefColumnCount();
        @Property int prefRowCount();
    }

    public UXTextArea(Environment env, TextArea wrappedObject) {
        super(env, wrappedObject);
    }

    public UXTextArea(Environment env, ClassEntity clazz) {
        super(env, clazz);
    }

    @Override
    public TextArea getWrappedObject() {
        return (TextArea) super.getWrappedObject();
    }

    @Signature
    public void __construct() {
        __wrappedObject = new TextArea();
    }

    @Signature
    public void __construct(String text) {
        __wrappedObject = new TextArea(text);
    }
}
